client/my-sites/site-settings/disconnect-site/index.jsx

```diff
@@ -16,27 +16,41 @@ import { getSelectedSite } from 'state/ui/selectors';
 import Main from 'components/main';
 import Placeholder from 'my-sites/site-settings/placeholder';
 import redirectNonJetpack from 'my-sites/site-settings/redirect-non-jetpack';
+import NavigationBackButton from 'my-sites/site-settings/navigation-back-button';
 import SkipSurvey from './skip-survey';
 
 class DisconnectSite extends Component {
+	getRoute() {
+		const { siteSlug } = this.props;
+
+		if ( siteSlug ) {
+			return '/settings/manage-connection/' + siteSlug;
+		}
+	}
+
 	render() {
 		const { site, translate } = this.props;
 
 		if ( ! site ) {
 			return <Placeholder />;
 		}
 		return (
-			<Main className="disconnect-site site-settings">
-				<DocumentHead title={ translate( 'Site Settings' ) } />
-				<FormattedHeader
-					headerText={ translate( 'Disconnect Site' ) }
-					subHeaderText={ translate(
-						'Tell us why you want to disconnect your site from WordPress.com.'
-					) }
-				/>
-				<Card className="disconnect-site__card"> </Card>
-				<SkipSurvey />
-			</Main>
+			<div>
+				<span className="disconnect-site__back-button-container">
+					<NavigationBackButton redirectRoute={ this.getRoute() } { ...this.props } />
```

Comment:
Do we really want to pass all props down to `<NavigationBackButton />`? It's always more readable and sustainable to be specific about which props you're passing down, rather than just passing them all. And that's even more valid in this case - I don't see any additional props necessary there, so maybe remove `{ ...this.props }` completely?